src/lib.rs

```diff
@@ -449,6 +563,18 @@ macro_rules! create_app {
                             .takes_value(true),
                     ),
             )
+            .subcommand(
+                SubCommand::with_name("describe-eif")
+                    .about("Returns information about the EIF found at a given path.")
+                    .arg(
+                        Arg::with_name("eif-path")
+                            .long("path")
+                            .short("p")
```

Comment:
Let's remove the short name as we do not have short names for the other commands.